### PR TITLE
doc: avoid multiple if statements

### PR DESCRIPTION
**Pull request by douglasffilho**
# What does this PR do?
- Create a doc ref that I believe must be useful for a guideline

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -562,6 +562,7 @@ keep the media queries as close to their relevant rule sets whenever possible.
 - 6.1 [Javascript Code Syntax](#javascript-syntax)
 - 6.2 [Variables](#variables) <br>
 - 6.3 [Descriptive validations (if)](#descriptive-validations) <br>
+- 6.4 [Avoid multiple ff's](#avoid-multiple-ifs) <br>
```
Comment (mauriciomutte) [inline README.md:565]: Small typo in "if"

**Review by mauriciomutte — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (douglasffilho) [reply README.md:565]: oops :grin:

**Review by mauriciomutte — APPROVED** [review]
Very nice tip, thank you so much!

File: README.md [hunk 2]
```diff
@@ -661,6 +662,48 @@ if (user.firstName && user.lastname) {
...
+
+Use an execution map instead a multiple if validations.
+
+**✅ Good:**
+
+```js
+const messagingChannels = {
+  whatspp: (message) => {
+    // send message to whatspp
+  },
```
Comment (lupy100) [inline README.md:675]:
```suggestion
  whatsapp: (message) => {
    // send message to whatsapp
  },
```

File: README.md [hunk 3]
```diff
@@ -661,6 +662,48 @@ if (user.firstName && user.lastname) {
...
+  // send message to whatsapp
+}
+
+const sendEmail = (message) => {
+  // send message to email
+}
+
+const sendMessage = (message, channel) => {
+  if (channel === 'whatsapp') {
+    sendWhatspp(message)
```
Comment (lupy100) [inline README.md:700]:
```suggestion
    sendWhatsapp(message)
```

**Review by lupy100 — CHANGES_REQUESTED** [review]

**Review by hick97 — APPROVED** [review]
nice! 🚀

**Review by lupy100 — APPROVED** [review]
Niceee

File: README.md [hunk 4]
```diff
@@ -661,6 +662,48 @@ if (user.firstName && user.lastname) {
 }
 ```
 
+### 6.4 Avoid multiple if's
```
Comment (lupy100) [inline README.md:665]: **suggestion:** add the anchor link please
```suggestion
<a name="avoid-multiple-ifs"></a>

### 6.4 Avoid multiple if's
```

**Review by lupy100 — CHANGES_REQUESTED** [review]
Sorry, I'd forgotten

File: README.md (hunk 4 above)
Comment (douglasffilho) [reply README.md:667]: Great!

**Review by lupy100 — APPROVED** [review]

**Pull request merged**